qcodes_contrib_drivers/drivers/Standa/Standa_10MWA168.py

```diff
@@ -0,0 +1,206 @@
+from qcodes import Instrument
+import ctypes
+import numpy as np
+import os
+import time
+
+
+class DeviceEnumeration(ctypes.BigEndianStructure):
+    pass
+
+
+class DeviceInformation(ctypes.Structure):
+    _fields_ = [
+        ("Manufacturer", ctypes.c_char * 5),
+        ("ManufacturerId", ctypes.c_char * 3),
+        ("ProductDescription", ctypes.c_char * 9),
+        ("Major", ctypes.c_uint),
+        ("Minor", ctypes.c_uint),
+        ("Release", ctypes.c_uint),
+    ]
+
+
+class GetPosition(ctypes.Structure):
+    _fields_ = [
+        ("Position", ctypes.c_int),
+        ("uPosition", ctypes.c_int),
+        ("EncPosition", ctypes.c_longlong),
+    ]
+
+
+class Status(ctypes.Structure):
+    _fields_ = [
+        ("MoveSts", ctypes.c_uint),
+        ("MvCmdSts", ctypes.c_uint),
+        ("PWRSts", ctypes.c_uint),
+        ("EncSts", ctypes.c_uint),
+        ("WindSts", ctypes.c_uint),
+        ("CurPosition", ctypes.c_int),
+        ("uCurPosition", ctypes.c_int),
+        ("EncPosition", ctypes.c_longlong),
+        ("CurSpeed", ctypes.c_int),
+        ("uCurSpeed", ctypes.c_int),
+        ("Ipwr", ctypes.c_int),
+        ("Upwr", ctypes.c_int),
+        ("Iusb", ctypes.c_int),
+        ("Uusb", ctypes.c_int),
+        ("CurT", ctypes.c_int),
+        ("Flags", ctypes.c_uint),
+        ("GPIOFlags", ctypes.c_uint),
+        ("CmdBufFreeSpace", ctypes.c_uint),
+    ]
+
+
+class libximc:
+    # TODO: use error check, implement wait for stop function from dll
+
+    # default dll path
+    _dll_path = 'C:\\Program Files\\XILab\\libximc.dll'
+
+    # success and error codes
+    _success_codes = {0: 'Ok'}
+    _error_codes = {-1: 'Error', -2: 'NotImplemented', -3: 'ValueError', -4: 'NoDevice'}
+
+    def __init__(self, dll_path=None, verbose=False):
+        # save attributes
+        self.verbose = verbose
+
+        # connect to the dll
+        current_path = os.getcwd()
+        os.chdir(os.path.dirname(self._dll_path))
+        self.dll = ctypes.windll.LoadLibrary(dll_path or self._dll_path)
+        os.chdir(current_path)
```

Comment:
plase wrap this in `try-finally` as here https://github.com/QCoDeS/Qcodes_contrib_drivers/pull/28